From 6bc253492c56e8035464d0976865d9b6fb317349 Mon Sep 17 00:00:00 2001
From: "Casper Ti. Vector" <CasperVector@gmail.com>
Date: Sat, 10 Sep 2016 10:27:01 +0800
Subject: [PATCH] sunpinyin-dictgen.mk.in: completely abandon (now defunct)
 google code.

Author:    Casper Ti. Vector <CasperVector@gmail.com>

The patch was imported from the github git server
(git@github.com:sunpinyin/sunpinyin) as of commit id
7435ccb64d623fa6fa4cdb5f5458845fda178b0e.
---
 src/sunpinyin-dictgen.mk.in | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/sunpinyin-dictgen.mk.in b/src/sunpinyin-dictgen.mk.in
index ee82c0c..e3e47fb 100644
--- a/src/sunpinyin-dictgen.mk.in
+++ b/src/sunpinyin-dictgen.mk.in
@@ -36,14 +36,14 @@ W3M = @W3M@
 ENDIANNESS = @ENDIANNESS@
 DATA_DIR = @DATADIR@/sunpinyin
 
-DL_LIST = https://open-gram.googlecode.com/git/
+DL_LIST = https://sourceforge.net/projects/open-gram/files/
 DL_HOST = http://heanet.dl.sourceforge.net
 DL_ROOT = ${DL_HOST}/open-gram
 DICT_PAT = 'dict\.utf8-[0-9]\+.tar.bz2'
 SLM_PAT = 'lm_sc\.3gm\.arpa-[0-9]\+.tar.bz2'
 
-DICT_AR = $(shell ${W3M} ${DL_LIST} | grep -o ${DICT_PAT} | sort | tail -n 1)
-SLM_AR = $(shell ${W3M} ${DL_LIST} | grep -o ${SLM_PAT} | sort | tail -n 1)
+DICT_AR = $(shell ${W3M} ${DL_LIST} | grep -o ${DICT_PAT} | sort -u | tail -n 1)
+SLM_AR = $(shell ${W3M} ${DL_LIST} | grep -o ${SLM_PAT} | sort -u | tail -n 1)
 
 all: install
 
-- 
1.9.0

